lib/active_interaction/filter.rb

```diff
@@ -90,15 +90,16 @@ def initialize(name, options = {}, &block)
     #   # => ActiveInteraction::InvalidDefaultError: example: 0
     #
     # @param value [Object]
+    # @param interaction [Base, nil]
     #
     # @return [Object]
     #
     # @raise (see #cast)
     # @raise (see #default)
-    def clean(value)
-      value = cast(value)
+    def clean(value, interaction)
```

Comment:
I still think that we're running the code in a `binding` or `context` regardless of what that context is. The fact that it's an interaction is secondary.